adoc/admin-crio-registries.adoc

```diff
@@ -0,0 +1,58 @@
+== Configuring container registries for {crio}
+
+Every registry related configuration needs to be done in the TOML file
```

Comment:
Are these changes to the TOML file applied automatically on save or do you need to somehow reload this config?

Reply:
Yes, I added a comment about that.